tests/test_fips.py

```diff
@@ -239,12 +239,21 @@ def test_gcrypt_binary(container_per_test: ContainerData) -> None:
 
     """
 
-    container_per_test.connection.check_output(
-        "zypper --gpg-auto-import-keys -n ref && zypper -n in gcc libgcrypt-devel && zypper -n clean && "
+    c = container_per_test.connection
+
+    c.check_output(
+        "zypper --gpg-auto-import-keys -n ref && "
```

Comment:
This shouldn't be necessary
```suggestion
        "zypper -n ref && "
```

Reply:
Yeah, that was a stupid mistake on my end… https://github.com/SUSE/BCI-tests/pull/697